common/process.h

```diff
@@ -53,6 +53,9 @@ struct ProcessStartInfo {
   // Command line, UTF-8 encoded.
   std::string command;
 
+  // Process startup working directory. If empty, uses parent's working dir.
+  std::string startup_dir;
```

Comment:
Should this be an absolute or relative path? If it is relative, is it interpreted relative to the parent's working dir? Does the directory need to exist?

Reply:
Full path. I'll add a comment. Docs say "The full path to the current directory for the process. The string can also specify a UNC path.

If this parameter is NULL, the new process will have the same current drive and directory as the calling process. (This feature is provided primarily for shells that need to start an application and specify its initial drive and working directory.)"